pf4j/src/test/java/org/pf4j/DefaultPluginFactoryTest.java

```diff
@@ -93,7 +93,7 @@ public void testCreateFailNotFound() {
      */
     @Test
     public void testCreateFailConstructor() {
-        PluginDescriptor pluginDescriptor = mock(PluginDescriptor.class);
+        PluginDescriptor pluginDescriptor = mock(DefaultPluginDescriptor.class);
```

Comment:
No need to change. I feel that it's from the refactoring action

Reply:
This test was not written by me but after I reviewed again the code I believe that original code is the right one. We need to test `DefaultPluginFactory` class and from this reason the author mock the code for `PluginDescriptor`. I don't know why don't use the real object `DefaultPluginDescriptor` without mock but this is another story.